config.yaml.sample

```diff
@@ -210,6 +217,10 @@ toi-prune:
     path: osm
     layer: all
     format: zip
+  # a reduced version of prev s3 entity
+  store:
+    layer: all
+    format: zip
```

Comment:
> it's very useful to have the configuration for the different operations separate, which allows us to handle production deploys and moments of transition more readily

What **moments of transition** mean here?

But different commands still operate on the same store, right? They can differ only like in `.alpha .beta .rc .release` where different stores might be used? In this circumstance why not having separate configuration like `config.alpha.yaml` for each maturity? Or you mean having something like:
```yaml
store:
  seed:
    type: directory
    name: tiles
    path: ../tiles
  prune-tiles-of-interest:
    type: s3
    name: tiles
    bucket: mapzen-tiles-dev-us-east
    reduced-redundancy: true
    date-prefix: 20170322
```